bin/templates/cordova/lib/emulator.js

```diff
@@ -238,13 +242,13 @@ module.exports.wait_for_emulator = function(num_running) {
  */
 module.exports.wait_for_boot = function(emulator_id) {
     var self = this;
-    return exec('adb -s ' + emulator_id + ' shell getprop init.svc.bootanim', os.tmpdir())
+    return exec('adb -s ' + emulator_id + ' shell ps', os.tmpdir())
     .then(function(output) {
-        if (output.match(/stopped/)) {
+        if (output.match(/android\.process\.acore/)) {
             return;
         } else {
             process.stdout.write('.');
-            return Q.delay(3000).then(function() {
+            return Q.delay(7000).then(function() {
```

Comment:
Wonder if doing this every 3 seconds might still be a good idea - why decrease this polling frequency?